### ISSUE TITLE
[TW-858] task info <filter>

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#893]
_David Patrick on 2010-08-19T14:37:31Z says:_ 

 Currently, *task info ID(s)* produces a full display of task information (or a series of such displays) in a very useful way. 
I think that it might be more useful still if

 task info < filter >

as in

 task info proj:foo due:eom

could be used to spit out an info listing report of all projects matching "foo", that were due by the end of the month, for example.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-08-19T14:37:31Z
Modified: 2014-02-09T02:06:40Z
```

**Comment by taskwarrior** [comment]
_Federico Hernandez on 2010-08-19T16:48:35Z says:_ 

 Well,

according to the man page

<pre>
info ID
Shows all data and metadata for the specified task.
</pre>

info wants a task ID, though it could be a sequence like (at)task info 4-6@.

So info does what it should. But perhaps the error msg should be changed.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-08-19T18:55:07Z says:_ 

 Functioning as designed.  You may want this on the Great New Parser Syntax Wishlist, because that conversation is coming up soon.

**Comment by taskwarrior** [comment]
_David Patrick on 2010-08-19T23:07:08Z says:_ 

 (changed from bug to feature request)

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2011-10-16T03:55:50Z says:_ 

 This one fell through the cracks.  It's been implemented in 2.0, but the ticket was forgotten.
